src/components/CategoryFilter.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+
+const bookCategories = [
```

Comment:
This array again is repeated here as well as in `BooksForm`. We should just have a separate file in a `utils` folder for our dummy books and this category array.